event-log/src/test/scala/io/renku/eventlog/init/DbInitSpec.scala

```diff
@@ -28,6 +28,7 @@ trait DbInitSpec extends InMemoryEventLogDb with EventLogDbMigrations with Befor
   self: Suite =>
 
   private val tablesToDropBeforeEachTest = List(
+    "event_delivery",
```

Comment:
Not sure if we use tablesToDropBeforeEachTest any more if we do we should maybe add the subscriber table